linux/jdk/debian/src/main/packaging/microsoft/11/debian/changelog

```diff
@@ -1,11 +1,11 @@
-msopenjdk-11 (11.0.19.0.0+7) STABLE; urgency=medium
+msopenjdk-11 (11.0.19-1) STABLE; urgency=medium
```

Comment:
Remind me again why the `-1` vs the Adoptium versioning scheme?

Reply:
This reflects the versioning that we have used since before we updated to the new linux installers creation

Reply:
OK, worth having a chat to Bruno and see if he wants to keep that versioning scheme or move to Adoptium's scheme

Reply:
Keeping this naming scheme for MSFT's Mariner distro